test/functional/eval/timer_spec.lua

```diff
@@ -165,7 +165,7 @@ describe('timers', function()
     command("call timer_start(10, 'MyHandler', {'repeat': -1})")
     eq(0,eval("g:val"))
     run(nil, nil, nil, load_adjust(50))
-    retry(nil, 5000, function()
+    retry(nil, nil, function()
       eq(3, eval("g:val"))
     end)
```

Comment:
what is the purpose of the run()? I think I should have removed it when I added the retry().

61e8adb25e6f37c3c9df